packages/cli/src/commands/build.ts

```diff
@@ -669,7 +669,16 @@ export async function runPackageJsonScript(
 
 async function linkOrCopy(existingPath: string, newPath: string) {
   try {
-    await fs.createLink(existingPath, newPath);
+    if (
+      newPath.endsWith('.nft.json') ||
+      newPath.endsWith('middleware-manifest.json')
```

Comment:
What's the significance of these two files? Are we not following symlinks for these two files but we do for the others?

Reply:
These are manipulated after they are copied/linked. https://github.com/vercel/vercel/issues/6975#issuecomment-965140034

Reply:
Let's make this clear with a comment here:

```
// These files will be mutated so they must be copied to avoid modifying the original files
```